modules/virt-enabling-bootsource-update.adoc

```diff
@@ -7,13 +7,16 @@
 [id="virt-enabling-bootsource-update_{context}"]
 = Enabling automatic boot source updates
 
-If you have pre-defined boot sources from {VirtProductName} 4.9, then you must manually opt them in to the automatic boot source updates. All pre-defined boot sources from {VirtProductName} 4.10 and later are automatically updated by default.
+If you have boot sources from {VirtProductName} 4.9 or earlier, you must manually turn on automatic updates for these boot sources. All boot sources in {VirtProductName} 4.10 and later are automatically updated by default.
+
+To enable automatic boot source imports and updates, set the `cdi.kubevirt.io/dataImportCron` field to `true` for each boot source you want to update automatically.
 
 .Procedure
 
-* Use the following command to apply the `dataImportCron` label to the data source:
+* To turn on automatic updates for a boot source, use the following command to apply the `dataImportCron` label to the data source:
 +
 [source,terminal]
 ----
-$ oc label --overwrite DataSource rhel8 -n openshift-virtualization-os-images cdi.kubevirt.io/dataImportCron=true
+$ oc label --overwrite DataSource rhel8 -n openshift-virtualization-os-images cdi.kubevirt.io/dataImportCron=true <1>
 ----
+<1> Specifying`true` turns on automatic updates for the `rhel8` boot source.
```

Comment:
Need a space before "true" so that the backticks render properly

Reply:
Thanks, done.